l10n_es_aeat_mod115/data/aeat_export_mod115_data.xml

```diff
@@ -0,0 +1,395 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2016 Antonio Espinosa <antonio.espinosa@tecnativa.com>
+     Copyright 2019 Tecnativa - Pedro M. Baeza
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <!--  SUB-115-01  -->
+    <record id="aeat_mod115_sub01_export_config" model="aeat.model.export.config">
+        <field name="name">Mod.115 2016 - Página</field>
+        <field name="model_number">115</field>
+        <field name="active" eval="False" />
+    </record>
+    <record id="aeat_mod115_sub01_export_line_01" model="aeat.model.export.config.line">
+        <field name="sequence">1</field>
+        <field name="export_config_id" ref="aeat_mod115_sub01_export_config" />
+        <field name="name">Constante: &lt;T</field>
+        <field name="fixed_value">&lt;T</field>
+        <field name="export_type">string</field>
+        <field name="size">2</field>
+        <field name="alignment">left</field>
+    </record>
+    <record id="aeat_mod115_sub01_export_line_02" model="aeat.model.export.config.line">
+        <field name="sequence">2</field>
+        <field name="export_config_id" ref="aeat_mod115_sub01_export_config" />
+        <field name="name">Constante: 115</field>
+        <field name="fixed_value">115</field>
+        <field name="export_type">string</field>
+        <field name="size">3</field>
+        <field name="alignment">left</field>
+    </record>
+    <record id="aeat_mod115_sub01_export_line_03" model="aeat.model.export.config.line">
+        <field name="sequence">3</field>
+        <field name="export_config_id" ref="aeat_mod115_sub01_export_config" />
+        <field name="name">Constante: 01</field>
+        <field name="fixed_value">01</field>
+        <field name="export_type">string</field>
+        <field name="size">2</field>
+        <field name="alignment">left</field>
+    </record>
+    <record id="aeat_mod115_sub01_export_line_04" model="aeat.model.export.config.line">
+        <field name="sequence">4</field>
+        <field name="export_config_id" ref="aeat_mod115_sub01_export_config" />
+        <field name="name">Constante: 000&gt;</field>
+        <field name="fixed_value">000&gt;</field>
+        <field name="export_type">string</field>
+        <field name="size">4</field>
+        <field name="alignment">left</field>
+    </record>
+    <record id="aeat_mod115_sub01_export_line_05" model="aeat.model.export.config.line">
+        <field name="sequence">5</field>
+        <field name="export_config_id" ref="aeat_mod115_sub01_export_config" />
+        <field name="name">Indicador de página complementaria</field>
+        <field name="expression">${object.type == 'C'}</field>
```

Comment:
Sale este error cuando exportas a BOE:

`AttributeError: 'l10n.es.aeat.mod115.report' object has no attribute 'type'`

Reply:
Si no me equivoco debería de ser `statement_type`

Reply:
Sí, y hay que hacer el script de migración que renombre el campo.

Reply:
Este es un ejemplo sobre el 303, sería hacerlo igual para este modelo
https://github.com/OCA/l10n-spain/blob/174cc7e471261c652207b009af4ec2da0c9be7b4/l10n_es_aeat_mod303/migrations/13.0.1.0.0/pre-migration.py#L4-L17

Reply:
I have commited the fix here: https://github.com/OCA/l10n-spain/commit/11a550d83e0cd292ead2ec211f7eddbeebf9ec05

But it seems I cannot create a PR to his branch. @acysos, could you cherry-pick that commit? Thanks.